openfeature/api.py

```diff
@@ -14,31 +14,52 @@
 
 _hooks: typing.List[Hook] = []
 
+_providers: typing.Dict[str, FeatureProvider] = {}
+
 
 def get_client(
-    name: typing.Optional[str] = None, version: typing.Optional[str] = None
+    domain: typing.Optional[str] = None, version: typing.Optional[str] = None
```

Comment:
this is basically a breaking change for anyone invoking it with keyword arguments.

Reply:
It's is. Are we ok with pushing this breaking change? Or should we have a deprecation period?

Reply:
Im totally fine with breaking changes, we just increase the minor version instead.

Reply:
Since this is a sub 1.0 SDK, I'd recommend just changing the title to `feat!...` (note the !) which will mark this as a breaking change in release-please. I think breaking changes are fine sub 1.0 and I don't think it's worth the maintenance burden to do non-breaking deprecations (just my 2c).